test

```diff
@@ -96,22 +96,40 @@ function cov_pass {
 
 	mkdir -p "$COVERDIR"
 
+	# convert COVERDIR to absolute path
+	# e2e needs absolute path to put its coverfiles
```

Comment:
why? should accept relative paths if possible. Use `filepath.IsAbs` in the go test code instead; if it isn't absolute, prepend with ".." via `filepath.Join("..", path)`